.gitignore

```diff
@@ -40,6 +40,7 @@ docs/source/backup
 # Testing
 .coverage
 cov*.xml
+test_data
```

Comment:
Since we expect test_data to be nested under tests (see below). This was not right in the first place so should not add it here (change the local path instead). Otherwise, this list will be too long.

Reply:
@shuo-zhou